packages/harpstrata/src/packages/slice-matrix.test.ts

```diff
@@ -0,0 +1,89 @@
+import { sliceMatrix } from './slice-matrix'
```

Comment:
It turned out later in https://github.com/js-jslog/harpguru/pull/112 that this was not the right place to be slicing up the matrices. It would be done in harpguru-core. This is just a record that this would normally not be accepted in to a merge to master.

Reply:
Raised as techdebt in #113 

The work done to make the app sizing dynamic (at #107) was used as a platform for additional speculative pull requests to ensure that this was the correct direction to start down.

Rather than addressing the finer details in this or subsequent pull request and potentially struggling to merge up to those subsequent branches I will prefer to record the issues in each merge request and resolve them afterwards.

This has a limit of course. Major issues will not be merged to master. I will need to find a way around that if it happens.